src/main/java/core/gui/model/UserColumnFactory.java

```diff
@@ -948,7 +948,48 @@ public IHOTableEntry getTableEntry(Player player, Player playerCompare) {
                 return new ColorLabelEntry(r, t, ColorLabelEntry.FG_STANDARD, ColorLabelEntry.BG_STANDARD, SwingConstants.RIGHT);
             }
         };
+        playerAdditionalArray[25] = new PlayerColumn(900, "ls.player.cost-to-convert-trainer-weak", "ls.player.cost-to-convert-trainer-weak.tooltip",  40) {
+            @Override
+            public IHOTableEntry getTableEntry(Player player, Player playerCompare) {
+                return getTrainerTransferEntry(player, 4);
+            }
+        };
+        playerAdditionalArray[26] = new PlayerColumn(901, "ls.player.cost-to-convert-trainer-inadequate", "ls.player.cost-to-convert-trainer-inadequate.tooltip", 40) {
+            @Override
+            public IHOTableEntry getTableEntry(Player player, Player playerCompare) {
+                return getTrainerTransferEntry(player, 5);
+            }
+        };
+        playerAdditionalArray[27] = new PlayerColumn(902, "ls.player.cost-to-convert-trainer-passable", "ls.player.cost-to-convert-trainer-passable.tooltip", 40) {
+            @Override
+            public IHOTableEntry getTableEntry(Player player, Player playerCompare) {
+                return getTrainerTransferEntry(player, 6);
+            }
+        };
+        playerAdditionalArray[28] = new PlayerColumn(903, "ls.player.cost-to-convert-trainer-solid", "ls.player.cost-to-convert-trainer-solid.tooltip", 40) {
+            @Override
+            public IHOTableEntry getTableEntry(Player player, Player playerCompare) {
+                return getTrainerTransferEntry(player, 7);
+            }
+        };
+        playerAdditionalArray[29] = new PlayerColumn(904, "ls.player.cost-to-convert-trainer-excellent", "ls.player.cost-to-convert-trainer-excellent.tooltip", 40) {
+            @Override
+            public IHOTableEntry getTableEntry(Player player, Player playerCompare) {
+                return getTrainerTransferEntry(player, 8);
+            }
+        };
 
         return playerAdditionalArray;
     }
+
+    private static IHOTableEntry getTrainerTransferEntry(Player player, int i) {
+        var costs = player.calculateCoachConversionCosts(i);
+        var string = "";
+        var swedishKrona = 0;
+        if (costs != null){
+            string = costs.toLocaleString();
+            swedishKrona = costs.getSwedishKrona();
```

Comment:
I must admit I had no idea that the Swedish Krona was used as the basis for that calculation!

Reply:
The formula was published using Euro, but the hattricks internally are using swedish krona in all xml files with a total unrealistic exchange course of 10 sk to 1 €;-)

Additionally all prices will be rounded to 100 sk respectively 10 €. 

These things i wanted to handle in one single utility class and no longer in several places with different approaches.

Reply:
Thanks, again I have learnt something!! 😊